bamboo/abort-if-not-terraform.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+#source .bamboo_env_vars || true
+if ! [[ $COMMIT_MESSAGE =~ terraform ]]; then
+  >&2 echo "******Skipping Terraform integration tests based on commit message: $COMMIT_MESSAGE"
+  exit 0
+fi
```

Comment:
Will require commit messages to include `terraform` in some shape or form for the time being. Thoughts?

Reply:
You might be able to avoid that 'mess' by doing it in the setup and having an environment variable flag available in the env vars fixture.      Just something to think about.